From 5a79b3533b3805cab5a538ea3f8c5c7ddd4d02a8 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 16 Jul 2018 20:52:14 +0800
Subject: [PATCH 4268/5242] MLK-18605-15 dt-bindings: display: panel: add
 'video-mode' prop for rm67191

commit  6d9d31d48fee26b0a51e9896c7ae08ec5e5ea491 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add a new property 'video-mode' binding for panel rm67191
which is used to specify a video data transfer mode.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 9dba8643d7b9c73a2b20ef517c79bb799f5ade3d)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../bindings/display/panel/raydium,rm67191.txt     |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/panel/raydium,rm67191.txt b/Documentation/devicetree/bindings/display/panel/raydium,rm67191.txt
index 18a57de..efda9db 100644
--- a/Documentation/devicetree/bindings/display/panel/raydium,rm67191.txt
+++ b/Documentation/devicetree/bindings/display/panel/raydium,rm67191.txt
@@ -17,6 +17,12 @@ Optional properties:
 - pinctrl-0		phandle to the pin settings for the reset pin
 - panel-width-mm:	physical panel width [mm]
 - panel-height-mm:	physical panel height [mm]
+- video-mode:		Video data transfer mode
+			must be <0>, <1> or <2> as
+			follows:
+			    <0>: Burst mode
+			    <1>: Non-burst mode with sync event
+			    <2>: Non-burst mode with sync pulse
 
 [1]: Documentation/devicetree/bindings/display/display-timing.txt
 
-- 
1.7.9.5

